### PR TITLE
Add gender attribute

### PR DESCRIPTION
**Pull request by ongweijie7**
Have resolved merge conflicts after @Creationsv2 's PR was merged in.

### CODE REVIEW

**Comment by ongweijie7** [comment]
Test cases have been updated to reflect new gender attribute

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -28,13 +29,15 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + PREFIX_DRUG_ALLERGY + "DRUG ALLERGIES "
+            + PREFIX_GENDER + "GENDER"
```
Comment (dawg420) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:32]:
```suggestion
            + PREFIX_GENDER + "GENDER "
```
space to make things look neater

**Review by dawg420 — COMMENTED** [review]
LGTM, thinking that we should just approve everything first then one shot solve all UI problems, cuz with each commit we might run into more UI issues.

**Review by Creationsv2 — APPROVED** [review]
LGTM.

**Pull request merged**